src-ts/node/events.ts

```diff
@@ -0,0 +1,5 @@
+export class EventEmitter {
+    constructor() {
+        throw new Error("TODO: Implement events.EventEmitter");
```

Comment:
why even mark this as TODO?

Reply:
We'll need it eventually, this way we get a clear message when we come across a concrete use case that actually needs it.